flake.nix

```diff
@@ -34,6 +34,7 @@
           clickhouse
           graphviz
           nodejs_22  # so basedpyright runs against it
+          bun
```

Comment:
should we have a separate docs/dev only shell?

Reply:
not needed now - CI is getting bun from elsehwere. would we want for local to get it from nix?

Locally I currently have bun from https://pixi.sh/latest/

```bash
pixi global install -c https://prefix.dev/brads-forge bun
```